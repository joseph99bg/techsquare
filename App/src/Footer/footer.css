.copyright-holder {
    background-color: #20AAEA;
    padding: 10px;
    display: flex;
    justify-content: center;
}
.copyright-holder span {
    color: #FFF;
    font-family: 'Orbitron';
    font-size: 14px;
}