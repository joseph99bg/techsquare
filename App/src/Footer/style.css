.copyright-holder {
    background-color: #20AAEA;
    padding: 10px;
    display: flex;
    justify-content: center;
    position: fixed;
    bottom: 0;
    width: 100%;
}
.copyright-holder span {
    color: #FFF;
    font-family: 'Orbitron';
    font-size: 14px;
}