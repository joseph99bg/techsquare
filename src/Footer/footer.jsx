import React from 'react';
import './footer.css';

function Footer() {
    return (
        <footer>
            <div className="copyright-holder">
                <span>
                    TechSquare Â© 2019. All rights reserved.
                </span>
            </div>
        </footer>
    )
}

export default Footer;