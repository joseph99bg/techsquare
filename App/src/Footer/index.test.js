import React from 'react'
import Footer from './index'
import renderer from 'react-test-renderer'

describe('Footer component', () => {
    test('should render correctly', () => {
        const component = renderer.create(
            <Footer />
        )
        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})