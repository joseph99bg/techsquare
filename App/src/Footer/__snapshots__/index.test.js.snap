// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render correctly 1`] = `
<footer>
  <div
    className="copyright-holder"
  >
    <span>
      TechSquare Â© 2019. All rights reserved.
    </span>
  </div>
</footer>
`;
